tests/test_ckan_to_frictionless.py

```diff
@@ -1,17 +1,22 @@
+# !/usr/bin/python
+# coding=utf-8
```

Comment:
Lines 1 and 2 were required to avoid an error caused on [line 140](https://github.com/frictionlessdata/frictionless-ckan-mapper/pull/20/files#diff-183798f142869cae35a36e4db32c64f1R140) that was due to not specifying the encoding.